README.md

```diff
@@ -0,0 +1,210 @@
+
+Take a look at the `test-cases/PR-provision-test.yaml` file. It contains a number of
+configurable test parameters, such as DB cluster instance types and AMI IDs.
+In this example, we're assuming that you have copied `test-cases/PR-provision-test.yaml`
+to `test-cases/your_config.yaml`.
+
+All the test run configurations are stored in `test-cases` directory.
+
+Important: Some tests use custom hardcoded operations due to their nature,
+so those tests won't honor what is set in `test-cases/your_config.yaml`.
+
+the available configuration options are listed in [configuration_options](./docs/configuration_options.md)
+
+## Different backends
+
+### Docker
+
+**NOTE:** for docker run work with monitoring stack, user should be part of sudo group,
+and setup with passwordless access, see https://unix.stackexchange.com/a/468417 for example on how to setup
+
+We can also enable running with scylla formal docker images:
+```bash
+# example of running specific docker version
+export SCT_SCYLLA_VERSION=5.2.1
+hydra run-test longevity_test.LongevityTest.test_custom_time --backend docker --config test-cases/PR-provision-test-docker.yaml
+```
+
+### AWS - Amazon Web Services
+
+Change your current working directory to this test suite base directory,
+then run test. Example command line:
+```bash
+hydra run-test longevity_test.LongevityTest.test_custom_time --backend aws --config test-cases/your_config.yaml
+```
+
+This command line is to run the test method ``test_custom_time``, in
+the class ``Longevitytest``, that lies inside the file ``longevity_test.py``,
+and the test will run using the AWS data defined in the branch ``eu_west_1``
+of ``data_dir/your_config.yaml``.
+
+
+###  GCE - Google Compute Engine
+
+In order to run tests using the GCE backend, you'll need:
+
+1. A GCE account
+
+2. `cp data_dir/scylla.yaml data_dir/your_config.yaml`
+
+3. Edit the configuration file (data_dir/your_config.yaml) to tweak values present
+   in the `gce:` session of that file. One of the values you might want to
+   tweak is the scylla yum repository used to install scylla on the CentOS 7 image.
+
+With that said and done, you can run your test using the command line::
+```bash
+hydra run-test longevity_test.LongevityTest.test_custom_time --backend gce --config test-cases/scylla-lmr.yaml
+```
+
+### Azure
+
+You can run your test using the command line:
+```bash
+hydra run-test longevity_test.LongevityTest.test_custom_time --backend azure --config test-cases/PR-provision-test.yaml
+```
+
+## Types of Tests
+### [Artifact tests](./docs/artifacts_test.md)
+### Longevity Tests (TODO: write explanation for them)
+### Upgrade Tests (TODO: write explanation for them)
+### Performance Tests (TODO: write explanation for them)
+### Features Tests (TODO: write explanation for them)
+### Manager Tests (TODO: write explanation for them)
+### [K8S Functional Tests](./docs/k8s-functional-test.md)
+### [Microbenchmarking Tests](./docs/microbenchmarking.md)
+
+
+## Test operations
```

Comment:
@roydahan 
this section is a bit unhelpful here, maybe this should move into a section describing how longevity tests works in general, with more details and maybe few diagram, monitoring screen shots ?